opsdroid/loader.py

```diff
@@ -101,9 +101,20 @@ def import_module(config):
                 continue
 
         if module_spec:
-            module = Loader.import_module_from_spec(module_spec)
-            _LOGGER.debug(_("Loaded %s: %s."), config["type"], config["module_path"])
-            return module
+            try:
+                module = Loader.import_module_from_spec(module_spec)
+            except Exception as e:
+                _LOGGER.error(
```

Comment:
Perhaps this should be 
```suggestion
                _LOGGER.error(_("The following exception was raised while importing %s %s"), config["type"], config["module_path"])
```
due to the i18n/i10n (never know which is which haha) we are doing?